docs/reference/cluster.asciidoc

```diff
@@ -6,23 +6,30 @@
 ["float",id="cluster-nodes"]
 == Node specification
 
-Most cluster level APIs allow to specify which nodes to execute on (for
-example, getting the node stats for a node). Nodes can be identified in
-the APIs either using their internal node id, the node name, address,
-custom attributes, or just the `_local` node receiving the request. For
-example, here are some sample executions of nodes info:
+Most cluster level APIs allow to specify nodes (for example, getting
```

Comment:
Can you add something to sentence to explain that specifying nodes limits the results to those nodes? At least, I think that is what specifying the node does here, but I'm no expect.